renderer/components/Rom/index.tsx

```diff
@@ -39,6 +41,22 @@ export const Rom = ({
   return (
     <button onContextMenu={handleContextMenu} className="w-full">
       <Menu id={MENU_ID}>
+        {rom?.info && (
+          <Item
+            id="add-to-backlog"
+            onClick={async () => {
+              const res = await fetch(`/api/hltb?game=${rom.info?.title}`);
+              const items: HLTGame[] = await res.json();
+
+              if (items.length > 0) {
```

Comment:
Hmm, I'm thinking we should still add it if it doesn't find a match, we just don't have the hours

Reply:
Ahhh yes this makes much more sense, ty!